### PR TITLE
Fiexes issue #4657

### LINKED ISSUE #4657: DevTools Color of ContextMenu Text
Screenshot (48)

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

## Fixed issues
Fixes #4657

### CODE REVIEW

**Comment by danwalmsley** [comment]
@workgroupengineering hey, how do we contact you outside of github? your doing a lot of great PRs and we would like to open up a channel of communication with you.. and github is a bit slow for that.

File: src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml [hunk 1]
```diff
@@ -12,6 +12,9 @@
     <StyleInclude Source="resm:Avalonia.Themes.Default.DefaultTheme.xaml?assembly=Avalonia.Themes.Default"/>
     <StyleInclude Source="resm:Avalonia.Themes.Default.Accents.BaseLight.xaml?assembly=Avalonia.Themes.Default"/>
     <StyleInclude Source="resm:Avalonia.Controls.DataGrid.Themes.Default.xaml?assembly=Avalonia.Controls.DataGrid"/>
+    <Style Selector="ContextMenu">
```
Comment (maxkatz6) [inline src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml:15]: Can you use more specific selector?
Maybe something like "DataGrid ContextMenu"

**Review by maxkatz6 — CHANGES_REQUESTED** [review]

**Comment by workgroupengineering** [comment]
> 
> 
> @workgroupengineering hey, how do we contact you outside of github? your doing a lot of great PRs and we would like to open up a channel of communication with you.. and github is a bit slow for that.

Hi @danwalmsley ,
this message had escaped me.

I would like it, only my english is really bad.

**Comment by grokys** [comment]
@workgroupengineering any idea why the context menu foreground is wrong in the devtools? Seems strange that we'd need to fix this in this manner.

**Comment by grokys** [comment]
> Hi @danwalmsley , this message had escaped me. I would like it, only my english is really bad.

@workgroupengineering that's no problem, but if we could talk about these features before you implement them, we may save everyone a lot of time ;)

Please, contact one of us on gitter and we'll try to set up a space where we can talk about your features!

**Comment by workgroupengineering** [comment]
> 
> 
> @workgroupengineering any idea why the context menu foreground is wrong in the devtools? Seems strange that we'd need to fix this in this manner.

probably by inheriting the color of the text from the main app theme.

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**